### ISSUE TITLE
Hard to find Yenta API Plugin

### ISSUE THREAD

**Issue by docmucki** [stapler/netbeans-stapler-plugin#6]
Hi,

seems the "Yenta Plugin" is needed to install.

https://github.com/jglick/yenta

Found the nbm file in release file on the plug-in portal

http://plugins.netbeans.org/plugin/43938/ 

Maybe it would be helpful to mention this in the readme.

**Comment by jglick** [comment]
I am struggling to get this thing updated; nbjavac is an obstacle. See other issues.

**Comment by jglick** [comment]
Yes. I publish a ZIP of the two plugin NBMs _plus_ the Yenta NBM. Needs to be more clearly documented indeed.

**Comment by mdealer** [comment]
> 
> 
> Yes. I publish a ZIP of the two plugin NBMs _plus_ the Yenta NBM. Needs to be more clearly documented indeed.

Publish where?

**Comment by jglick** [comment]
Well, _was_ on plugins.netbeans.org, but that seems to be dead. Sigh. I will need to spend some time figuring out how to properly publish plugins these days.

**Comment by jglick** [comment]
(see #4)

**Comment by jglick** [comment]
also https://github.com/jglick/yenta/pull/1

**Comment by timboudreau** [comment]
FWIW, I publish builds of it as an NBM [here](https://timboudreau.com/modules).

But changing the `groupId` to something that can be published into Maven Central and doing that is the more permanent solution.

**Comment by piotrhoppeintel** [comment]
https://web.archive.org/web/20170625030640/http://plugins.netbeans.org/plugin/43938/

**Comment by piotrhoppe** [comment]
https://web.archive.org/web/20170625030640/http://plugins.netbeans.org/plugin/43938/ - old version of plugin and yenta
